### PR TITLE
Updated lower and upper

### PR DESCRIPTION
**Pull request by shellhub**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/lower.py [hunk 1]
```diff
@@ -19,9 +17,7 @@ def lower(word: str) -> str:
     # converting to ascii value int value and checking to see if char is a capital
     # letter if it is a capital letter it is getting shift by 32 which makes it a lower
     # case letter
-    return "".join(
-        chr(ord(char) + 32) if 65 <= ord(char) <= 90 else char for char in word
-    )
+    return "".join(chr(ord(char) + 32) if "A" <= char <= "Z" else char for char in word)
```
Comment (mateuszz0000) [inline strings/lower.py:20]: I am not sure but in my opinion this change causes a decrease of algorithm speed. Is comparing strings is not faster than comparing ints - in `ord()` case? @cclauss Your knowledge is need here! :)

Comment (cclauss) [reply strings/lower.py:20]: Anyone up for creating a timeit benchmark?

**Comment by shellhub** [comment]
I test it using `timeit`
``` python
def lower_faster(word: str = "HelloWorld I LOVE PYTHON") -> str:
    return "".join(
        chr(ord(char) + 32) if 'A' <= char <= 'Z' else char for char in word
    )


def lower_slower(word: str = "HelloWorld I LOVE PYTHON") -> str:
    return "".join(
        chr(ord(char) + 32) if 65 <= ord(char) <= 90 else char for char in word
    )


def benchmark() -> None:
    import timeit
    print(timeit.timeit("lower_slower()", setup="from __main__ import lower_slower", number=1000))
    print(timeit.timeit("lower_faster()", setup="from __main__ import lower_faster", number=1000))


if __name__ == "__main__":

    benchmark()

```
output:
```
0.005637370000000003
0.005095732000000002
```

**Comment by cclauss** [comment]
Please remove `number=1000` and re-benchmark.

**Comment by shellhub** [comment]
> Please remove `number=1000` and re-benchmark.

like this below ?
``` python
print(timeit.timeit("lower_slower()", setup="from __main__ import lower_slower"))
```

**Comment by shellhub** [comment]
After I removed.
``` python
def lower_faster(word: str = "HelloWorld I LOVE PYTHON") -> str:
    return "".join(
        chr(ord(char) + 32) if 'A' <= char <= 'Z' else char for char in word
    )


def lower_slower(word: str = "HelloWorld I LOVE PYTHON") -> str:
    return "".join(
        chr(ord(char) + 32) if 65 <= ord(char) <= 90 else char for char in word
    )


def benchmark() -> None:
    import timeit
    print(timeit.timeit("lower_slower()", setup="from __main__ import lower_slower"))
    print(timeit.timeit("lower_faster()", setup="from __main__ import lower_faster"))


if __name__ == "__main__":

    benchmark()

```
output
```
4.604586689
4.297654392999999
```

**Review by cclauss — APPROVED** [review]
It is always nice to have a benchmark to verify performance.

**Comment by shellhub** [comment]
You are my Python teacher. I learn so much from you. Thanks a lot 👍

**Pull request merged**